pkgs/by-name/do/doas-keepenv/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  bash,
+  coreutils,
+  doas,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "doas-keepenv";
+  version = "1.0-1";
+
+  src = fetchTarball {
+    url = "https://github.com/stas-badzi/doas-keepenv/archive/refs/tags/1.0.tar.gz";
+    sha256 = "sha256:15xh3dgw78v6mfgkqv6mphpw0bxxbg7jqrpkb4y5151a9xjc9962";
+  };
+
+  buildInputs = [
+    bash
+    coreutils
+    doas
+  ];
+
+  nativeBuildInputs = [
+    coreutils
+    makeWrapper
+  ];
```

bash and coreutils are not required to be explicitly added.
```suggestion
  buildInputs = [
    doas
  ];

  nativeBuildInputs = [
    makeWrapper
  ];
```